viadot/sources/databricks.py

```diff
@@ -0,0 +1,350 @@
+import json
+from pandas import DataFrame
```

Comment:
This import is confusing as you're dealing with two types of DataFrames in this file. This should be imported as eg. `pd.DataFrame` to always know which DataFrame class we're dealing with.